graal/backends/core/analyzers/scancode.py

```diff
@@ -62,9 +69,62 @@ def analyze(self, **kwargs):
             subprocess._cleanup()
 
         licenses_raw = json.loads(msg)
-        if 'files' not in licenses_raw:
-            return result
+        if 'files' in licenses_raw:
+            result['licenses'] = licenses_raw['files'][0]['licenses']
+
+        return result
+
+    def __analyze_scancode_cli(self, file_paths):
+        """Add information about license using scancode-cli
+
+        :param file_paths: file paths
+        :returns result: dict of the results of the analysis
+        """
+
+        result = {'files': []}
+
+        try:
+            cmd_scancli = ['python3', self.exec_path]
+            cmd_scancli.extend(file_paths)
+            msg = subprocess.check_output(cmd_scancli).decode("utf-8")
+        except subprocess.CalledProcessError as e:
+            raise GraalError(cause="Scancode failed at %s, %s" % (file_paths,
+                                                                  e.output.decode("utf-8")))
+        finally:
+            subprocess._cleanup()
+
+        output_content = ''
+        outputs_json = []
+        for line in msg.split('\n'):
+            if line == '':
+                if output_content:
+                    output_json = json.loads(output_content)[1:]
+                    outputs_json.append(output_json)
+                    output_content = ''
+            else:
+                output_content += line
+
+        if output_content:
+            output_json = json.loads(output_content)[1:]
+            outputs_json.append(output_json)
+
+        for output_json in outputs_json:
+            file_info = output_json[0]['files'][0]
+            result['files'].append(file_info)
+
+        return result
+
+    def analyze(self, **kwargs):
+        """Add information about license
+
+        :param file_path: file path
```

Comment:
This param doesn't seem to be clear enough (since it is valid only for scancode). Maybe we could add another param (`file_paths`) and add some text to explain when `file_path` and `file_paths` are valid. What do you think ?

Reply:
Thanks for pointing that out. Yes, it's not clear as of now.
I was thinking something like:
```
:param file_path: file path ( in case of scancode )
:param file_paths: file paths ( in case of scancode_cli for concurrent execution on files )
```

What do you think?

Reply:
It looks OK! Let's go for this solution and if in the future we see it isn't clear we will change it (maybe scancode cli could be another analyzer).

Reply:
Thanks for the quick response :)